sites/partners/src/components/listings/PaperListingForm/sections/ListingIntro.tsx

```diff
@@ -109,6 +139,32 @@ const ListingIntro = (props: ListingIntroProps) => {
             />
           </Grid.Cell>
         </Grid.Row>
+        <Grid.Row columns={1}>
+          <Grid.Cell>
+            <FieldGroup
+              name="hasEbllClearance"
+              type="radio"
+              register={register}
+              groupLabel={t("listings.hasEbllClearanceTitle")}
+              fields={[
+                {
+                  id: "ebllYes",
+                  label: t("t.yes"),
+                  value: YesNoEnum.yes,
+                  defaultChecked: listing?.hasEbllClearance === true,
+                },
+                {
+                  id: "ebllNo",
+                  label: t("t.no"),
+                  value: YesNoEnum.no,
+                  defaultChecked: listing?.hasEbllClearance === false,
```

Comment:
question: We have been switching to have radio selectors have a default value selected. I'm wondering if we should have that be the case for this? I see that we don't have a default value on the backend so maybe it's good to not have a default one set here.